### PR TITLE
Glossary: allow html entities like &nbsp; or &copy;

### PR DESCRIPTION
**Pull request by dpfaffenbauer**
Solves error 

`An exception has been thrown during the rendering of a template ("Warning: DOMDocumentFragment::appendXML(): Entity: line 1: parser error : Entity 'nbsp' not defined").`

### CODE REVIEW

**Review by dvesh3 — COMMENTED** [review]
It throws another error on apply the change "_An exception has been thrown during the rendering of a template ("Warning: DOMDocumentFragment::appendXML(): Entity: line 2: parser error : chunk is not well balanced"). File: /var/www/html/templates/areas/wysiwyg/view.html.twig Line: 10_"

Could please try replacing the line 135 with 
`$text = htmlspecialchars($parentNode->text(), ENT_QUOTES | ENT_HTML5);`

**Comment by dpfaffenbauer** [comment]
What did you try?

**Comment by dvesh3** [comment]
> What did you try?

I tried adding both `&nbsp;` and `&copy;` in the wysiwyg editable source.

**Comment by dpfaffenbauer** [comment]
@dvesh3 interesting, doesn't happen for me... replaced the line though

**Comment by dvesh3** [comment]
> > What did you try?
> 
> I tried adding both `&nbsp;` and `&copy;` in the wysiwyg editable source.

Actually, It even works without any change, [check test line]( https://demo.pimcore.fun/en/More-Stuff/Developers-Corner/Glossary). Please validate your html source again.

**Comment by dpfaffenbauer** [comment]
Screenshot 2022-06-22 at 11 52 47

**Comment by dvesh3** [comment]
> Screenshot 2022-06-22 at 11 52 47

someone is messing up with the demo 😄

**Comment by dvesh3** [comment]
@dpfaffenbauer this is what i tried:
image

image

**Comment by dpfaffenbauer** [comment]
@dvesh3 I'll write a test and make it more reproducable

**Comment by dpfaffenbauer** [comment]
@dvesh3 added a test, with that HTML it breaks.

File: bundles/CoreBundle/Resources/config/templating.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ services:
     # GLOSSARY
 
     Pimcore\Tool\Glossary\Processor:
-        public: false
+        public: true #For Tests
```
Comment (jdreesen) [inline bundles/CoreBundle/Resources/config/templating.yaml]: Not really cool, to make it `public` only for tests, no? Doesn't Codeception has a better way for this? Like the Symfony test container?

Comment (dpfaffenbauer) [reply bundles/CoreBundle/Resources/config/templating.yaml]: no idea

Comment (dvesh3) [reply bundles/CoreBundle/Resources/config/templating.yaml]: maybe this helps https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Testing/Testing_Services.html#page_Testing-Symfony-Services

Comment (dpfaffenbauer) [reply bundles/CoreBundle/Resources/config/templating.yaml]: done

**Comment by dvesh3** [comment]
@dpfaffenbauer thanks!

**Pull request merged**